_posts/en/newsletters/2019-12-28-newsletter.md

```diff
@@ -252,9 +252,8 @@ onchain transaction fees.  This makes Loop almost completely trustless.
   carve-out][topic cpfp carve out] mempool policy, added initial support
   for [BIP158][]-style [compact block filters][topic compact block
   filters] (currently RPC only), improved security by deprecating or
-  removing support for protocols such as [BIP37][] bloom filters and
-  [BIP70][] payment requests.  It also switches GUI users to bech32
-  addresses by default.
+  removing support for protocols such as [BIP37][] bloom filters.
```

Comment:
How about we make this (diff from the orig)

```diff
      for [BIP158][]-style [compact block filters][topic compact block
-     filters] (currently RPC only), and improved security by deprecating or
-     removing support for protocols such as [BIP37][] bloom filters and
-     [BIP70][] payment requests.  It also switches GUI users to bech32
+     filters] (currently RPC only), improved security by
+     disabling protocols such as [BIP37][] bloom filters and
+     [BIP70][] payment requests by default.  It also switches GUI users to bech32
```

Reply:
Sure, amended with your patch.